docker-compose.yaml

```diff
@@ -280,9 +289,8 @@ services:
     <<: *inference-service-cpu
     container_name: chat-completions
     profiles: [chat_completions_mistralrs_cpu]
-    build:
-      context: https://github.com/EricLBuehler/mistral.rs.git
-      dockerfile: Dockerfile
+    image: ghcr.io/ericlbuehler/mistral.rs:cpu-0.4
+    platform: linux/amd64
```

Comment:
I believe we want to have support for arm64 in here as well (at least, building from source)